examples/dev-mashup/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "local-mashup",
   "version": "1.0.0",
   "description": "Simple local mashup",
-  "main": "index.html",
+  "main": "index.js",
```

Comment:
Do you know why the main is needed? I do not find the main section in https://github.com/qlik-oss/nebula.js/blob/master/commands/create/templates/mashup/_package.json.

Reply:
It's for npm or for the compiler/build tool to know which file is the entry point of the application. So no real use for it in this case. We could also remove it completely but it will still default to `index.js` which is why I decided to keep it (no strong opinion about it though).

Reply:
Very good to know. Then it is fine to keep it.